utils/oscap_docker_python/get_cve_input.py

```diff
@@ -45,7 +45,7 @@ class getInputCVE(object):
     def __init__(self, fs_dest, DEBUG=False):
         ''' Simple init declaration '''
         self.dest = fs_dest
-        self.DEBUG = DEBUG
+        self.DEBUG = True
```

Comment:
Is this intentional or just a oversight? I think if we set this to true, we don't have to check for it (`if self.DEBUG:`) further in the code. I can't find any other place in the code where this variable is set.